README.md

```diff
@@ -36,10 +36,14 @@ As mentioned in the command output, don't forget to add the installation directo
 
     2. manually from git, or download from https://www.rust-lang.org/install.html (the 'rustc' source download behind the 'source' link is the right one).
 
-2. Set the ```RUST_SRC_PATH``` environment variable to point to the 'src' dir in the Rust source installation
+    Ensure to download sources for any rust version you have installed (e.g. a common configuration is to have the `stable` and many `nightly` installed).
```

Comment:
> to have the `stable` and many `nightly` installed

I don't understand what you mean here.
But, I'm not a native English speaker so I'm not sure.

Reply:
neither am I :-) what I tried to explain here is perphaps redundant but Ii happened to me yesterday; imagine the following workflow:

- user downloads stable rust using `rustup`
- user downloads sources and installs `racer`
- at some point in the future, user downloads a nightly build because (f.e.) they want to use Rocket 
- `racer` partially stops working because user forgot to also download the nightly build sources

([just happened to me yesterday](https://github.com/racer-rust/emacs-racer/issues/93#issuecomment-393495176))

Do you think it's worth such a note of it ends up being more confusing? Any suggestion on how to rephrase more clearly? e.g. simply with `Remember to download sources after installing a nightly build too`

Reply:
Ah I understand, thanks.
Then my suggestion is:
**Note** 
If you want to use racer with multiple release channels(we can use 3 release channels, stable, beta and nightly), you have to download Rust sourcecode for each relase channels.
e.g.(rustup case)
You have to run `rustup component add rust-src` after `rustup toolchain add nightly`.

Reply:
ack, thank you!